addons/mail/static/src/components/message/message_tests.js

```diff
@@ -1212,6 +1212,66 @@ QUnit.test('rendering of tracked field with change of value from empty to string
     );
 });
 
+QUnit.test('basic rendering of tracking value (monetary type)', async function (assert) {
+    assert.expect(8);
+
+    await this.start();
+    this.env.session.currencies = {1: {symbol: '$', position: 'before'}};
```

Comment:
proper env override in test should be like this (I didn't test if I miss type anything, but it's for the general idea)
```suggestion
    await this.start({
        env: {
            session: {
                currencies: { 1: { symbol: '$', position: 'before' } },
            },
        },
    });
```
Your solution works by luck but if the currency was used in start it would be updated too late.